core/src/main/java/com/novoda/noplayer/exoplayer/forwarder/ExoPlayerDrmSessionEventListener.java

```diff
@@ -0,0 +1,43 @@
+package com.novoda.noplayer.exoplayer.forwarder;
+
+import com.google.android.exoplayer2.drm.DefaultDrmSessionManager;
+
+import java.util.List;
+import java.util.concurrent.CopyOnWriteArrayList;
+
+public class ExoPlayerDrmSessionEventListener implements DefaultDrmSessionManager.EventListener {
```

Comment:
These are the internal listeners that are attached to either `exo-player` or `media-player`. The ones you are thinking of are the ones exposed on the public api.